src/utils/metrics.py

```
@@ -7,12 +7,30 @@
 
 """Convenience wrappers around the ``empyrical`` statistics library."""
 
+import importlib
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)